scripts/validate-version-strategy.mjs

```
@@ -0,0 +1,149 @@
+async function main() {
+  try {
+    await validateVersionStrategy()
+    validateDistTagStrategy()
+    console.log('\n🎉 Version strategy validation completed successfully')
+  } catch (error) {
+    console.error('💥 Validation failed:', error)
+    process.exit(1)
+  }
+}
+
+if (import.meta.url === `file://${process.argv[1]}`) {
```

ESLint

[error] 'process' is not defined.

(no-undef)